kmr_mg-ud-test.conllu

```diff
@@ -6659,20 +6659,20 @@
 15	.	.	PUNCT	sent	_	3	punct	_	_
 
 # sent_id = wiki:wikibank.vislcg.txt:54:1468
-# text = Di 647an Iran, Lîbya hat dagir kirin û dagir kirina Anatoliya jî dest pê kir.
+# text = Di 647an Iran, Lîbya hat dagirkirin û dagir kirina Anatoliya jî dest pê kir.
 1	Di	di	ADP	pr	AdpType=Prep	2	case	_	_
 2	647an	647	NUM	num	Case=Acc|Number=Plur|NumType=Card	7	nmod	_	_
 3	Iran	îran	PROPN	np	Case=Nom|Definite=Def|Gender=Fem|Number=Sing	7	nsubj	_	SpaceAfter=No
 4	,	,	PUNCT	cm	_	5	punct	_	_
 5	Lîbya	Lîbya	PROPN	np	Case=Nom|Gender=Masc|Number=Sing	3	conj	_	_
 6	hat	hatin	AUX	vblex	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	7	aux	_	_
-7	dagir kirin	dagirkirin	VERB	vblex	VerbForm=Inf	0	root	_	_
+7	dagirkirin	dagir kirin	VERB	vblex	VerbForm=Inf	0	root	_	_
```

Comment:
The verb is infinitive is "dagir kirin". [See](https://ku.wiktionary.org/wiki/dagir_kirin)